conda.recipe/meta.yaml

```diff
@@ -1,10 +1,13 @@
+{% set data = load_setup_py_data() %}
+
 package:
   name: bokeh
-  version: 'nightly'
+  version: {{ data['version'] }}
+
+source:
+  path: ..
```

Comment:
Are you sure you want o use `path` here... some time ago I was told that `path` was going into "deprecation" and that we should use `git_url` instead, not sure if this is the actual recommendation right now, @msarahan thoughts?